src/embedded/app/restapi.lua

```diff
@@ -1,8 +1,9 @@
 local restapi = {
 	incubator = nil,
-	configurator = require("configurator")
+	configurator = require("configurator"),
 }
 
+wifi = require("wifiinit")
```

Comment:
cuantas veces se llama a la funcion connect de wifi?  podes copiar aqui un log desde que arranca ... segun veo wifi = require("wifiinit") va a llamar a connect y el configurador tambien... se llamaria dos veces connect ... eso no es lo que quisiera ... 
Si se llama dos veces como creo que sucede quisiera que por favor desasgas este cambio para poder aprobar la pull request. Y lo dejes tal y como estaba cuando lo aprobé. y tal como sugerí en el correo anterior.

Reply:
Running
�[0;32mI (4335) phy_init: phy_version 4791,2c4672b,Dec 20 2023,16:06:06�[0m
I (4415) wifi:mode : sta (94:b9:7e:ce:11:14) + softAP (94:b9:7e:ce:11:15)
I (4415) wifi:enable tsf
I (4425) wifi:Total power save buffer number: 16
I (4425) wifi:Init max length of beacon: 752/752
I (4425) wifi:Init max length of beacon: 752/752
I (4485) wifi:ap channel adjust o:11,2 n:1,1
I (4485) wifi:new:<1,0>, old:<11,2>, ap:<1,1>, sta:<1,0>, prof:11
I (4485) wifi:state: init -> auth (b0)
I (4495) wifi:state: auth -> assoc (0)
I (4505) wifi:state: assoc -> run (10)
W (4505) wifi:[ADDBA]rx delba, code:39, delete tid:0
W (4515) wifi:[ADDBA]rx delba, code:39, delete tid:0
�[34m[TRACE 1970-01-01 00:00:03 DST:0]�[0m  : Conecting to WiFi access point...
�[0;32mI (4595) gpio: GPIO[21]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4595) gpio: GPIO[22]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 �[0m
I (4615) wifi:connected with ChinaNet-POLLO2_0, aid = 2, channel 1, BW20, bssid = 74:3c:18:12:0c:60
I (4615) wifi:security: WPA2-PSK, phy: bgn, rssi: -62
I (4635) wifi:pm start, type: 1

I (4635) wifi:dp: 1, bi: 102400, li: 3, scale listen interval from 307200 us to 307200 us
I (4635) wifi:AP's beacon interval = 102400 us, DTIM period = 1
�[0;32mI (4655) gpio: GPIO[2]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4655) gpio: GPIO[13]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4665) gpio: GPIO[15]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4685) gpio: GPIO[26]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4685) gpio: GPIO[33]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4705) gpio: GPIO[34]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 �[0m
�[0;32mI (4705) gpio: GPIO[35]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 �[0m
starting server .. fyi maxtemp 37.8
> �0;32mI (5005) esp_netif_lwip: DHCP server started on interface WIFI_AP_DEF with IP: 192.168.16.10�[0m
�[34m[TRACE 1970-01-01 00:00:04 DST:0]�[0m  : conecction to AP ChinaNet-POLLO2_0 established!
�[34m[TRACE 1970-01-01 00:00:04 DST:0]�[0m  : Waiting for IP address...
I (5165) wifi:<ba-add>idx:0 (ifx:0, 74:3c:18:12:0c:60), tid:0, ssn:3, winSize:64
�[34m[TRACE 1970-01-01 00:00:05 DST:0]�[0m  : NodeMCU IP config: 192.168.1.6 netmask 255.255.255.0 gw192.168.11
�[34m[TRACE 1970-01-01 00:00:05 DST:0]�[0m  : Startup will resume momentarily, you have 3 seconds to abort.
�[34m[TRACE 1970-01-01 00:00:05 DST:0]�[0m  : Waiting...
5     hora vieja
5    hora nueva
�[0;32mI (6245) esp_netif_handlers: sta ip: 192.168.1.6, mask: 255.255.255.0, gw: 192.168.1.1�[0m